src/ucentral-client/platform/brcm-sonic/Makefile

```diff
@@ -13,7 +13,7 @@ netlink/netlink.full.a:
 
 %.o: %.c
 ifdef PLATFORM_REVISION
-	gcc -c -o $@ ${CFLAGS} -I ./ -I ../../include -D PLATFORM_REVISION='"$(PLATFORM_REVISION)"' $^
+	gcc -c -g -o $@ ${CFLAGS} -I ./ -I ../../include -D PLATFORM_REVISION='"$(PLATFORM_REVISION)"' $^
```

Comment:
why do we need to explicitly generate debug info for each type of build? This flag has to be controlled from higher call tree perspective, i think;
Especially since these flags are unconditionally changed for existing platform